tests/CMakeLists.txt

```diff
@@ -217,22 +217,25 @@ endif()
 target_link_libraries(arangodbtests
   arangoserver
   rocksdb
+  fuerte
   gtest
 )
 
 target_include_directories(arangodbtests PRIVATE
   ${INCLUDE_DIRECTORIES}
 )
 
-target_include_directories(arangodbtests SYSTEM PRIVATE
-  ${V8_INCLUDE_DIR}
-)
+
+if(MSVC)
+  target_compile_options(arangodbtests PRIVATE /WX /D_WINSOCK_DEPRECATED_NO_WARNINGS)
+endif()
 
 # add these includes as system includes because otherwise
 # the compiler will emit warnings for fakeit.hpp
 target_include_directories(arangodbtests SYSTEM PRIVATE
+  ${V8_INCLUDE_DIR}
   ${CMAKE_SOURCE_DIR}/3rdParty/fakeit-gtest
-  ${CMAKE_SOURCE_DIR}/3rdParty/rocksdb/${ARANGO_ROCKSDB_VERSION}/third-party/gtest-1.7.0/fused-src
+  ${CMAKE_SOURCE_DIR}/3rdParty/rocksdb/${ARANGO_ROCKSDB_VERSION}/third-party/gtest-1.8.1/fused-src
```

Comment:
Are we upgrading a 3rdParty-helper-lib of 3rdParty-lib? I think we wanted to add gtest as direct 3rdParty library so it is independent of rocksdb.

Reply:
yes, its been a separate PR: https://github.com/arangodb/arangodb/pull/9193 however, I didn't want to fix windows warnings in the old gtest, since they differ, so this PR requires the other.